packages/jazz-run/src/index.ts

```diff
@@ -39,6 +41,78 @@ const accountCommand = Command.make("account").pipe(
   Command.withSubcommands([createAccountCommand]),
 );
 
+const ownerOption = Options.text("owner").pipe(
+  Options.withAlias("o"),
+  Options.withDescription("The owner of the group"),
+  Options.withDefault(process.env.JAZZ_WORKER_ACCOUNT),
+);
+
+const ownerSecretOption = Options.text("secret").pipe(
+  Options.withAlias("s"),
+  Options.withDescription("The secret of the owner"),
+  Options.withDefault(process.env.JAZZ_WORKER_SECRET),
+);
+
+const createGroupCommand = Command.make(
+  "create",
+  {
+    owner: ownerOption,
+    ownerSecret: ownerSecretOption,
```

Comment:
Shall we use an env variable instead?

Passing a secret via CLI doesn't seem safe considering that it will be stored in the shell history.